ClosedXML_Tests/Excel/Cells/XLCellTests.cs

```diff
@@ -256,6 +256,172 @@ public void IsEmpty6()
             Assert.AreEqual(expected, actual);
         }
 
+        [Test]
+        public void IsEmpty_Comment()
+        {
+            IXLWorksheet ws = new XLWorkbook().Worksheets.Add("Sheet1");
+            IXLCell cell = ws.Cell(1, 1);
+            cell.Comment.AddText("comment");
+            bool actual = cell.IsEmpty();
+            bool expected = false;
+            Assert.AreEqual(expected, actual);
```

Comment:
`Assert.False(cell.IsEmpty());` would be shorter, though it's just a matter of preference.